<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-TIC5DM4" failures="1" tests="1" name="Test1" time="7.884" errors="0" timestamp="22 Aug 2024 07:51:19 GMT">
  <testcase classname="package1.tc1" name="f on [tc1 passed=0 failed=0 skipped=0]" time="7.66">
    <failure type="java.lang.AssertionError" message="expected [Demo Web Shop] but found [Demo Web Shops]">
      <![CDATA[java.lang.AssertionError: expected [Demo Web Shop] but found [Demo Web Shops]
	at package1.tc1.f(tc1.java:19)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- f on [tc1 passed=0 failed=0 skipped=0] -->
</testsuite> <!-- Test1 -->
